platform_tools/direct_access.rst

```diff
@@ -59,4 +59,5 @@ The DE Africa STAC endpoint is:
 https://explorer.digitalearth.africa/stac/
 
 For more information, see the
-`SpatioTemporal Asset Catalog <https://stacspec.org/>`_ site.
+`SpatioTemporal Asset Catalog <https://stacspec.org/>`_ site and the
+`DE Africa odc-stac example<odc_stac.ipynb>`_.
```

Comment:
The hyperlink isn't displaying correctly on the website -- it might be that you need a space between `example` and `<odc_stac.ipynb>`.

![Screen Shot 2022-07-06 at 10 09 28 am](https://user-images.githubusercontent.com/25995927/177436685-ea8cc351-25a0-4300-8dee-da45fd7fd48d.png)